src/app/modals/modals.component.scss

```diff
@@ -0,0 +1,7 @@
+.modal.modal-inverse {
```

Comment:
Only thing we need to be aware of is when we don't include these files via `styleUrls` we should scope them to the component we want. In our case it's our demo modals component with the selector `app-modals`. Wrap all the styles in 

app-modals {
    .....
    .....
}